sci-physics/tauola/tauola-1.1.8-r1.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit fortran-2
+
+MY_PN="TAUOLA"
+MY_P=${MY_PN}.${PV}
+
+DESCRIPTION="Tau decay Monte Carlo generator"
+HOMEPAGE="http://tauolapp.web.cern.ch/"
+SRC_URI="https://tauolapp.web.cern.ch/resources/${MY_P}/${MY_P}.tar.gz"
+S=${WORKDIR}/${MY_PN}
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+hepmc3 +lhapdf doc examples hepmc pythia tau-spinner"
+REQUIRED_USE=" || ( hepmc hepmc3 ) tau-spinner? ( lhapdf )"
+
+RDEPEND="
+	hepmc? ( sci-physics/hepmc:2=[-cm(-),gev(+)] )
+	hepmc3? ( sci-physics/hepmc:3=[-cm(-),gev(+)] )
+	pythia? ( sci-physics/pythia:8= )
+	lhapdf? ( sci-physics/lhapdf )
+"
+DEPEND="${RDEPEND}
+	doc? (
+		app-text/doxygen[dot]
+		app-text/ghostscript-gpl
+		app-text/texlive
+	)
+"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-1.1.8-tau-spinner-makefile-install.patch
+)
+
+src_configure() {
+	econf \
+		$(use_with lhapdf) \
+		$(use_with tau-spinner) \
+		$(use_with pythia pythia8 "${EPREFIX}/usr") \
+		$(use_with hepmc hepmc "${EPREFIX}/usr") \
+		$(use_with hepmc3 hepmc3 "${EPREFIX}/usr") \
+		--without-mc-tester
+	# weird autoconf + Makefile
+	cat << EOF >> make.inc || die
```

Comment:
```suggestion
	cat << -EOF >> make.inc || die
```

Reply:
Apparently there should be no space between `<<` and `-EOF`

Reply:
aah yes, my bad

Reply:
I'll get started disentangling+rebasing the commit and then test the PRs again.